test/postgres.spec.ts

```diff
@@ -9,6 +9,7 @@ describe('postgres', () => {
 
   before(() => {
     database = Knex({
+      searchPath: ['public', 'test'],
```

Comment:
if somebody knows how to multiple these tests so that we have searchpath = array, searchpath = string and no search path? I tested that manually, no clue about all those JS testing stuff

Reply:
@codeclown Thoughts?